shrine/.metadata

```diff
@@ -0,0 +1,8 @@
+# This file tracks properties of this Flutter project.
```

Comment:
Are you sure? It actually says in the file that it should be version-controlled.

Reply:
I'm not sure! Could we get more info from the Flutter team proper?

Reply:
After asking around, it looks like this file stores the ID of the version of the flutter SDK used with the project so that later versions can know which changes to make when someone upgrades.

Reply:
Cool. Like a kind of lock file. In that case, it might make sense for this repo but not for the codelabs which will def have a requirement to run `create`.